lib/python/rose/apps/rose_ana.py

```diff
@@ -76,8 +77,7 @@ def create(self):
         "If the databaste doesn't exist, create it."
         conn = self.get_conn()
         # This table stores the individual comparisons
-        conn.execute(
-            """
+        sql_statement = """
             CREATE TABLE IF NOT EXISTS comparisons (
             app_task TEXT,
```

Comment:
This is a little like magic to me, comment what this SQL command is actually trying to do?